index.html

```diff
@@ -1,21 +1,325 @@
+                        </figure>
+                        <p class="subheader2format text">
+                            <span>The Power of</span>
+                            <span>Wellness Shots</span>
+                            <span></span>
+                        </p>
+                        <p class="paragraphtext1">
+                            <span></span>
+                        </p>
+                    </div>
+                </div>
+
+            </article>
+            <article class="ViewAllButton">
+                <div class="centre">
+                    <a href="viewing.html" class="AButton LButton">View All</a>
+                </div>
+            </article>
+        </article>
+
+        <article class="WhiteArea">
+            <div class="row-flexdisplay">
+                <figure>
+                    <img src="img/boxof-bottles.webp" class="WhitePicture" alt="a box of bottled assorted juices">
+                </figure>
+                <div>
+                    <p class="subheader2format text WhiteHeader">
+                        <span>Discover Our</span>
+                        <span>Special Cleanse Kit</span>
+                        <span></span>
+                    </p>
+                    <p class="paragraphtext2">
+                        <span></span>
+                    </p>
+                    <article class="TryItButton">
+                        <div class="centre">
+                            <a href="#" class="AButton WButton">Try It</a>
+                        </div>
+                </div>
+            </div>
+        </article>
+
+        <p class="subheader2format text GetJuiceBarHeader">
+            <span>Get Juice Bar</span>
+            <span></span>
+        </p>
+
+        <article class="display-3items-flexrow">
+            <div class="display-3numbers">
+                    <p class="Circle">01</p>
+                    <p class="subheader2format">
+                        <span>Local Delivery</span>
+                    </p>
+                    <p class="paragraphtext3">
+                        <span></span>
+                    </p>
+            </div>
+
+            <div class="display-3numbers">
+                    <p class="Circle">02</p>
+                    <p class="subheader2format">
+                        <span>In Store Pick Up</span>
+                    </p>
+                    <p class="paragraphtext3">
+                        <span></span>
+                    </p>
+            </div>
+
+            <div class="display-3numbers">
+                    <p class="Circle">03</p>
+                    <p class="subheader2format">
+                        <span>Curbside Pick Up</span>
+                    </p>
+                    <p class="paragraphtext3">
+                        <span></span>
+                    </p>
+            </div>
+        </article>
+
+        <p class="gap1"></p>
```

Comment:
I would rather put apply the margin to the next tag (i.e. `<article class="FollowUs">`). For example: `<article class="FollowUs gap1">`. This is much cleaner.